Dockerfile

```diff
@@ -1,19 +1,20 @@
-FROM golang:1.16beta1
+FROM golang:1.19
 
 ENV LANG=en_US.UTF-8 \
     LANGUAGE=en_US:en \
     LC_ALL=en_US.UTF-8
 
-RUN apt-get update -q && apt-get install -y zip ruby ruby-dev rpm locales && \
-  go get github.com/kardianos/govendor && \
-  go get github.com/buildkite/github-release && \
-  gem install --no-ri --no-rdoc rake fpm package_cloud && \
-  echo "en_US UTF-8" > /etc/locale.gen && \
+RUN apt-get update -q && \
+  apt-get install -y zip ruby ruby-dev rpm locales
+
+RUN go install github.com/buildkite/github-release@latest
+
+RUN gem install --no-document rake fpm package_cloud
+
+RUN echo "en_US UTF-8" > /etc/locale.gen && \
```

Comment:
From memory, the reason to do all of these in one line is because docker makes a "layer" for each RUN. Which basically saves all the files. I don't think it matters much when your adding files though, it's more of a thing if you add and remove a file in a single line you can you avoid it being saved at all. So I think the readability improvement here is still worth it.

Reply:
Yeah — I unpacked it because I wanted the very slow `apt-get` stuff cached as a layer while I iterated on the subsequent `go install` and `gem install` commands, and then didn't both repacking them because the trade-offs didn't seem worthwhile.